api/lib/infrastructure/repositories/sco-certification-candidate-repository.js

```diff
@@ -19,15 +19,15 @@ const addNonEnrolledCandidatesToSession = async function ({ sessionId, scoCertif
     .filter((candidate) => !alreadyEnrolledCandidateOrganizationLearnerIds.includes(candidate.organizationLearnerId))
     .map(scoCandidateToDTO);
 
-  const addedCandidateIds = await knex
+  const addedCandidates = await knex
     .batchInsert('certification-candidates', candidatesToBeEnrolledDTOs)
-    .returning(knex.raw('id as "certificationCandidateId", \'??\' as type', [SubscriptionTypes.CORE]));
+    .returning(knex.raw('id as "certificationCandidateId"'));
 
-  // addedCandidateIds.forEach((candidate) => {
-  //   candidate.type = SubscriptionTypes.CORE;
-  // });
+  addedCandidates.forEach((candidate) => {
+    candidate.type = SubscriptionTypes.CORE;
+  });
 
-  await knex.batchInsert('certification-subscriptions', addedCandidateIds);
+  await knex.batchInsert('certification-subscriptions', addedCandidates);
```

Comment:
Ça m'a légèrement titillé de ne pas avoir réussi à le faire uniquement avec knex, du coup en implémentant cela les tests passent :

```suggestion
await knex
    .batchInsert('certification-candidates', candidatesToBeEnrolledDTOs)
    .returning([knex.raw('id as "certificationCandidateId"'), knex.raw(`'${SubscriptionTypes.CORE}' as type`)])
    .then(async function (partialSubscriptions) {
      await knex.batchInsert('certification-subscriptions', partialSubscriptions);
    });
```